app/views/users/_response_templates.html.erb

```diff
@@ -60,6 +55,21 @@
         <%= f.text_area :content, value: content, class: "crayons-textfield" %>
       </div>
 
+      <% if current_user.has_trusted_role? && @response_template.new_record? %>
+        <div class="crayons-field">
+          <div>
+            <%= f.radio_button :type_of, "personal_comment",
+                               checked: @response_template.type_of != "mod_comment",
+                               class: "crayons-radio", role: "radio" %>
+            <%= label_tag "response_template_type_of_personal_comment", "Personal" %>
+
+            <%= f.radio_button :type_of, "mod_comment",
+                               class: "crayons-radio", role: "radio" %>
+            <%= label_tag "response_template_type_of_mod_comment", "Trusted User" %>
+          </div>
```

Comment:
```suggestion
          <fieldset>
            <legend class="screen-reader-only">Template type</legend>
            <%= f.radio_button :type_of, "personal_comment",
                               checked: @response_template.type_of != "mod_comment",
                               class: "crayons-radio", role: "radio" %>
            <%= label_tag "response_template_type_of_personal_comment", "Personal" %>

            <%= f.radio_button :type_of, "mod_comment",
                               class: "crayons-radio", role: "radio" %>
            <%= label_tag "response_template_type_of_mod_comment", "Trusted User" %>
          </fieldset>
```
It's a bit of a nitpick, but it would be nice semantics if we can group these together in a fieldset with an accessible legend. I haven't tested locally, but I'm hopeful that using `fieldset` in place of `div` shouldn't have any side effects, and the `screen-reader-only` class should make sure the legend is visually hidden (I'm not sure what the design requirements are but this should keep it consistent with how it is just now)